charts/posthog/tests/_statsd.yaml

```diff
@@ -0,0 +1,82 @@
+# Tests charts/posthog/templates/_statsd.tpl
+suite: _statsd.tpl template
+templates:
+  - templates/test-templates.yaml
```

Comment:
What are these test templates? Can we not just check that env vars are in the env array?

Reply:
I e.g. would this work, and be more concrete:

```
    asserts:
      - contains:
          path: spec.template.spec.containers[0].env
          content:
            name: STATSD_HOST
            value: RELEASE-NAME-posthog-prometheus-statsd-exporter
```

etc.

Reply:
(we can then also verify that the envs are added to the appropriate deployments)

Reply:
Sure, I can add that